tests/opentelemetry/test_context.py

```diff
@@ -3,12 +3,24 @@
 import time
 
 import opentelemetry
+
+# from opentelemetry.baggage import clear
```

Comment:
I recommend removing these changes, opening a separate draft PR if you want to keep track of them.